model/gen-ai/spans.yaml

```diff
@@ -175,6 +175,22 @@ groups:
         requirement_level:
           conditionally_required: If not default (443).
 
+  - id: span.gen_ai.gcp.client
+    type: span
+    extends: trace.gen_ai.client.common_attributes
+    stability: development
+    brief: Semantic Conventions for model inference spans when calling Google systems.
+    span_kind: client
+    note: |
+      `gen_ai.system` MUST be set to `"gcp.gen_ai"`, `"gcp.vertex_ai"`, or `"gcp.gemini"`.
+    attributes:
+     - ref: gcp.gen_ai.operation.config
+       requirement_level: opt_in
+     - ref: gcp.client.service
```

Comment:
How about use `members` as `type` for this? Like https://github.com/open-telemetry/semantic-conventions/blob/main/model/gen-ai/registry.yaml#L255-L282

Reply:
Is this a comment about `gcp.gen_ai.operation.config` or about `gcp.client.service`?

It looks like `members` defines an `enum` type. However `gcp.gen_ai.operation.config` is not an enum. The `gcp.cllient.service` is not defined by this PR (though this PR references it).

Is it valid to change the type where it is referenced?

Reply:
I don't think that should be turned into an enum, because that would create a lot of toil whenever a new GCP API is launched and would fall quickly out of date. Additionally, I expect that most GCP client libraries will continue to be auto generated and therefore no one will think to update the value in OTel SemConv. Manually written GCP client libraries like Gen AI SDK are the exception rather than the rule.